website/server/controllers/top-level/payments/iap.js

```diff
@@ -54,12 +54,27 @@ api.iapAndroidVerify = {
       userId: user._id,
     });
 
-    await payments.buyGems({
-      user,
-      paymentMethod: 'IAP GooglePlay',
-      amount: 5.25,
-      headers: req.headers,
-    });
+    let amount = 0;
+
+    switch (receiptObj.productId) {
+      case 'com.habitrpg.android.habitica.iap.4gems':
+        amount = 1;
+        break;
+      case 'com.habitrpg.android.habitica.iap.20gems':
+      case 'com.habitrpg.android.habitica.iap.21gems':
+        amount = 5.25;
+        break;
+      case 'com.habitrpg.android.habitica.iap.42gems':
+        amount = 10.5;
+        break;
+      case 'com.habitrpg.android.habitica.iap.84gems':
+        amount = 21;
+        break;
+    }
+
+    if (amount === 0) throw new Error('INVALID_ITEM_PURCHASED');
+
+    await payments.buyGems({user, paymentMethod: 'IAP GooglePlay', amount, headers: req.headers}); // eslint-disable-line babel/no-await-in-loop
```

Comment:
the eslint comment should not be necessary, also can you split the object on multiple lines as it was before?